encryption_works.md

```diff
@@ -339,53 +339,43 @@ Snowden was right to be cautious and to insist that he check Gellman's new PGP f
-* Each person should create a new pseudonymous Jabber account. One way to do this is to go to [https://register.jabber.org/](https://register.jabber.org/) in Iceweasel. Since Tails makes all Internet traffic go over Tor, this is effectively making an anonymous Jabber account.
-* Each person should open Pidgin and configure it to use their new Jabber account and create a new OTR key. Everyone should add each other to their buddy lists and start OTR sessions with eachother. Since everyone is in the same room, this is the perfect time to compare fingerprints and verify the identity of all parties so that you'll able to communicate securely over the Internet in the future.
-* Each person should create a new pseudonymous email address as well. Some email providers, such as Gmail, make it very difficult to create new accounts while using Tor and staying anonymous, so find another email provider to use instead. Make sure your email provider supports IMAP (so you can use a desktop email client) over SSL (so your email client uses encryption when communicating with the email srever). If everyone chooses the same email provider, sending emails between accounts should never leave that email server, which reduces the metadata about your email usage available to anyone conducting dragnet surveillance of the Internet.
-* Each person should generate a new PGP key for their email address. Like with disk encryption, it's important to choose a strong passphrase when generating a PGP key.
-* The PGP-enabled email client that comes with Tails is called Claws Mail. Each person should configure Claws Mail to use their new email address, and then email a copy of their public key to all other people in the room.
-* Each person should import everyone else's public key into their keyring, and should manually verify the PGP fingerprints. Don't skip this step. In the end, each person should have a keyring containing signed keys of each other person.
+Journalists rely on their notes, and so the idea of an operating system that saves nothing between sessions may sound optimal. But Tails has the optional feature of persistence -- saving files in an encrypted volume which are accessible across separate sessions. This is useful for files you're working on, and encryption keys you use with Tails. (It's generally inadvisable to move keys off a Tails machine -- you should keep them compartmentalized and away from your regular computer operating system.)
 
-If a malicious attacker physically steals your Tails USB stick, modifies it, and gives it back, he can compromise all of the security of Tails. For this reason, it's important to keep your USB stick with you at all times.
```

Comment:
Why delete this? Important caveat. Let's put back in. Also can say as another disclaimer, something like: "While Tails prevents many endpoint attacks, like all security tools, it is not 100% impervious. Recently, researchers have shown how attackers can implant malware in the BIOS - or the brains - of the computer, away from the hard drive. Using this type of sophisticated technique may allow Tails to be compromised[link]. However, it's unclear how widespread this type of hacking technique is or if it's particularly effective in the wild."